moveit_ros/move_group/src/default_capabilities/move_action_capability.cpp

```diff
@@ -107,17 +107,6 @@ void MoveGroupMoveAction::executeMoveCallbackPlanAndExecute(const moveit_msgs::M
   {
     planning_scene_monitor::LockedPlanningSceneRO lscene(context_->planning_scene_monitor_);
     const moveit::core::RobotState& current_state = lscene->getCurrentState();
-
-    // check to see if the desired constraints are already met
-    for (std::size_t i = 0; i < goal->request.goal_constraints.size(); ++i)
-      if (lscene->isStateConstrained(current_state,
```

Comment:
I think this's a bug anyway since it looks for the first constraint that is satisfied, rather than all of them, right?

Reply:
Yes, you are right! I didn't check this...